client/src/api/api.ts

```diff
@@ -1,52 +1,28 @@
-import { AnimeControllerApi, Configuration, ConfigurationControllerApi } from "./generated";
-import { Anime, Config, Episode, Page } from "../domain";
+import { api as generatedApi } from "./generated";
+import { isRejectedWithValue, Middleware, MiddlewareAPI } from "@reduxjs/toolkit";
+import { notAuthorized } from "../auth";
+import store from "../store";
 
-// FIXME need to replace any type
-// eslint-disable-next-line
-const catchError = (response: Response): any => {
-    console.log("error: ", JSON.stringify(response));
-    if (response.status === 401) {
-        const redirect = encodeURIComponent(window.location.pathname);
-        window.location.href = `/login?redirect=${redirect}`;
-    }
-    //show "something happened
-};
-
-// openapi description specifies 8080 port
-// this configuration overrides port
-const controllerConfiguration = new Configuration({
-    basePath: "",
+export const api = generatedApi.enhanceEndpoints({
+    endpoints: {},
```

Comment:
looks like all endpoints without caching, what about providesTags/invalidatesTags?

Reply:
Created an issue to provide tags.  Cache indeed works.